src/main/java/org/jabref/gui/search/GlobalSearchResultDialogViewModel.java

```diff
@@ -29,4 +35,17 @@ public BibDatabaseContext getSearchDatabaseContext() {
     public BooleanProperty keepOnTop() {
         return this.keepOnTop;
     }
+
+    public int searchWindowHeight() {
+        return this.searchWindowHeight.get();
+    }
+
+    public int searchWindowWidth() {
+        return this.searchWindowWidth.get();
+    }
+
+    public void updateWindowSize(int height, int width) {
+        searchPreferences.setSearchWindowHeight(height);
+        searchPreferences.setSearchWindowWidth(width);
+    }
```

Comment:
Why don't you directly bind the properties in SearchPreferences to the view? The window size should not be handled in the view model, which must be completly indepentend of the view.